pkg/elemental/store/elemental-store/actions.ts

```diff
@@ -0,0 +1,19 @@
+// import { handleSpoofedRequest } from '@shell/plugins/dashboard-store/actions';
```

Comment:
Not on this PR, you are right... I think I will use it for the second PR (work that I am doing now). May I keep this now 👼 and promise to clean on the next PR if don't really use it or do you prefer that I clean up the code right now?
I am good with both since the store implementation is quite simple (afaik)